backend/app/api/endpoints/deep_research.py

```
@@ -300,11 +276,49 @@ async def stream_deep_research_result(
+            logger.error("[DEEP_RESEARCH] Stream error: %s", e)
+            yield _format_sse_event(
+                "response.error",
+                {
+                    "code": "stream_error",
+                    "message": str(e),
+                    "status_code": e.status_code,
+                },
+            )
+
+        except Exception as e:
+            logger.exception("[DEEP_RESEARCH] Unexpected stream error: %s", e)
```

Ruff (0.15.2)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)